trie/triedb/pathdb/asyncnodebuffer.go

```diff
@@ -18,12 +18,13 @@ import (
 
 var _ trienodebuffer = &asyncnodebuffer{}
 
-// asyncnodebuffer implement trienodebuffer interface, and aysnc the nodecache
+// asyncnodebuffer implement trienodebuffer interface, and async the nodecache
 // to disk.
 type asyncnodebuffer struct {
 	mux        sync.RWMutex
 	current    *nodecache
 	background *nodecache
+	isFlushing atomic.Bool
```

Comment:
I think need a `stop` flag to stop flush when call waitAndStopFlush

Reply:
yes, fixed